trio/_core/_multierror.py

```diff
@@ -460,10 +514,14 @@ def trio_show_traceback(self, etype, value, tb, tb_offset=None):
 
     assert sys.excepthook is apport_python_hook.apport_excepthook
 
-    def replacement_excepthook(etype, value, tb):
-        sys.stderr.write("".join(format_exception(etype, value, tb)))
+    def replacement_excepthook(
+        etype: type[BaseException], value: BaseException, tb: TracebackType | None
+    ) -> None:
+        # This does work, it's an overloaded function
+        sys.stderr.write("".join(format_exception(etype, value, tb)))  # type: ignore[arg-type]
 
     fake_sys = ModuleType("trio_fake_sys")
     fake_sys.__dict__.update(sys.__dict__)
-    fake_sys.__excepthook__ = replacement_excepthook  # type: ignore
+    # Fake does not have __excepthook__ attribute, but we are about to replace real sys
```

Comment:
It does have an `__excepthook__` attribute.... just type checkers don't recognize `fake_sys.__dict__.update(sys.__dict__)`.

Reply:
Changed comment in [5ad1cab](https://github.com/python-trio/trio/pull/2742/commits/5ad1cab84968e3ce6384e95abb433802e0448c43)